### PR TITLE
Apply formatters and set up CI

### PR DESCRIPTION
**Pull request by zigaLuksic**
The CI will only trigger for push events for now.
Flake8 finds a couple hundred issues, so I'm not sure we want to enable it just yet.

### CODE REVIEW

File: .github/workflows/ci_action.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.CHECKOUT_BRANCH }}
+    
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```
Comment (batic) [inline .github/workflows/ci_action.yml:19]: This part might be problematic. Some of the examples we have, could potentially only work on some old versions (of sh-py/eo-learn/Python in general). We've been discussing that perhaps we'd rather require that each example comes with specs for env).

File: core/TimeLapse.ipynb [hunk 2]
```diff
@@ -137,18 +137,10 @@
    "outputs": [],
    "source": [
     "# define workflow\n",
-    "workflow = LinearWorkflow(\n",
-    "    add_data_task,\n",
-    "    filter_task\n",
-    ")\n",
+    "workflow = LinearWorkflow(add_data_task, filter_task)\n",
```
Comment (batic) [inline core/TimeLapse.ipynb:140]: 🙈

**Review by batic — COMMENTED** [review]
Apart from the comment regarding Python version used for linting, I agree. Note: I only skimmed the rest of the files, as I trust "black" and "isort" did things correctly. Let's discuss about versions in the next days.

File: .github/workflows/ci_action.yml (hunk 1 above)
Comment (zigaLuksic) [reply .github/workflows/ci_action.yml:19]: The good part about `black` and `isort` is that they don't require any kind of package installation (apart from themselves). Therefore it shouldn't really matter what python version the formatters are installed in. Until we beef it up (unlikely to happen soon) I don't think it should be much of an issue.

File: core/TimeLapse.ipynb (hunk 2 above)
Comment (zigaLuksic) [reply core/TimeLapse.ipynb:140]: :see_no_evil:

File: .github/workflows/ci_action.yml (hunk 1 above)
Comment (batic) [reply .github/workflows/ci_action.yml:19]: Good. Then the PR has my blessing ;)

**Pull request merged**